client/stylesheets/_modal.scss

```diff
@@ -184,8 +184,9 @@
 	margin: 0;
 }
 
-.modOfferRequestPageOneUl li h2, .modOfferRequestPageTwo li h2, .modOfferRequestPageThree h2, .modAddListingDiv h2, .modFeedbackInitialDiv h2, .modReportDiv h2 {
-	color: $color-spaceblue !important;
+.modOfferRequestPageOneUl li h2, .modOfferRequestPageTwo li h2, .modOfferRequestPageThree h2, 
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `modOfferRequestPageOneUl` should be written in lowercase with hyphens<br>Selector `modOfferRequestPageTwo` should be written in lowercase with hyphens<br>Selector `modOfferRequestPageThree` should be written in lowercase with hyphens<br>Selector `modAddListingDiv` should be written in lowercase with hyphens<br>Selector `modFeedbackInitialDiv` should be written in lowercase with hyphens<br>Selector `modReportDiv` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line<br>Line contains trailing whitespace